app/commands/decidim/time_tracker/admin/update_assignee.rb

```diff
@@ -30,8 +33,8 @@ def update_assignee!
           Decidim.traceability.update!(
             @assignee,
             @user,
-            status: :accepted,
-            invited_by_user: @user
+            invited_by_user: @user, # TODO: Why?
```

Comment:
the user the first user is used for the traceability log
this one is for the assignee. In any case I don't think we should update the `invited_by_user` after creating, so safe to remove

Reply:
>     * In the assignee list, maybe it would simple to just remove accept/reject icons and add a select with the status
>       ![image](https://user-images.githubusercontent.com/1401520/97551266-0d7cab00-19d3-11eb-8f28-5734c9b5399c.png)

I think we can create an _enhancement_ issue for this change, what do you think?

>     * Breadcrump font size looks tiny
>       ![image](https://user-images.githubusercontent.com/1401520/97551356-300ec400-19d3-11eb-83bd-5f0d8295a7a2.png)

I tried to include custom css for the admin but couldn't get it to work, can you help me with that? 🙏🏽  Thanks!

Reply:
ok, let's separate it